src/System.Text.Encoding/tests/ASCIIEncoding/ASCIIEncodingGetMaxByteCount.cs

```diff
@@ -2,27 +2,19 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Collections.Generic;
 using Xunit;
 
 namespace System.Text.Tests
 {
     public class ASCIIEncodingGetMaxByteCount
     {
-        private static readonly RandomDataGenerator s_randomDataGenerator = new RandomDataGenerator();
```

Comment:
It's not actually random: the call sites are passing in specific seeds, so the same results are generated each time.  If the goal is to have randomized input, we should be strategic about what those are and actually allow for a variety of truly randomized inputs.  Otherwise, there's no point in cluttering up the tests will all of these calls to generate random data when it's not actually random... we should instead hardcode the values we care about, which could very well be output that these calls are generating, since it's effectively hardcoded anyway.